src/subscript/welltest_dpds/welltest_dpds.py

```diff
@@ -0,0 +1,472 @@
+#!/usr/bin/env python
+
+import sys
+import os
+import argparse
+import numpy as np
+from ecl.summary import EclSum
+
+
+DESCRIPTION = """
+Script to extract simulated welltest results from simulator output.
+
+Required summary vectors in sim deck:
+  -days,
+  -wbhp:well_name,
+  -wwpr:well_name,
+  -wopr:well_name if --phase == OIL
+  -wgpr:well_name if --phase == GAS
+
+Outputs the following files according to naming convention:
+  outputdirectory/key_outfilesuffix.csv
+  - dpds_lag1 eg superpositioned time derivative of pressure lag 1
+  - dpds_lag2 eg superpositioned time derivative of pressure lag 2
+  - sspt superpositioned time
+  - wbhp vs cum time
+  - wwpr vs cum time
+  - wopr vs cum time if --phase == OIL
+  - wgpr vs cum time if --phase == GAS
+
```

Comment:
This yields a RST warning: /home/berland/projects/subscript/src/subscript/welltest_dpds/welltest_dpds.py:docstring of subscript.welltest_dpds.welltest_dpds.DESCRIPTION:17: WARNING: Definition list ends without a blank line; unexpected unindent.